cmd/swarm/feeds_test.go

```diff
@@ -162,4 +162,37 @@ func TestCLIFeedUpdate(t *testing.T) {
 	if !bytes.Equal(data, retrieved) {
 		t.Fatalf("Received %s, expected %s", retrieved, data)
 	}
+
+	// test publishing a manifest for a different user
+	flags = []string{
+		"--bzzapi", srv.URL,
+		"--bzzaccount", pkFileName,
+		"feed", "create",
+		"--topic", topic.Hex(),
+		"--user", "0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", // different user
+	}
+
+	log.Info(fmt.Sprintf("Publishing manifest with 'swarm feed create' for a different user"))
```

Comment:
Why are we allowing this, really?

Reply:
Also please fix the log call, no need for fmt.Sprintf here.

Reply:
> Why are we allowing this, really?

We're not allowing it, that's the point. The thing is the CLI does not really give you an error when you try.

The problem is that if you create a manifest for user A and then try to sign an update for a feed based on that manifest but signing as B, the CLI did not give you an error. Now it will.

Reply:
This is a different aspect. When using `--bzzaccount` + `--user`, `--user` takes precedence. I acknowledge this can be confusing, but it is not what this PR is about.

The scope of this PR is adding an error message to someone trying to update someone else's already existing manifest. The CLI was silently failing in this circumstance. Now it will tell you.

Reply:
Ok, that means that the above also should not be allowed, but it will be fixed in a different PR. Or should it be discussed?

Reply:
So I found out the source of my confusion was that I ran:

`build/swarm_test --bzzapi http://localhost:8500 feed create`

and got:

`Fatal: bzzaccount option is required but not set; check your config file, command line or environment variables`

Suggesting that an account is needed to generate the manifest. But apparently this is the error you get if you don't fulfill the requirements of the _subcommand_ properly (for example here the missing `--user`). When I add `--user` it completes as it should, regardless of `--bzzaccount`. Maybe we could sort out these discrepancies?

Reply:
You must provide either `--bzzaccount` or `--user`. The user address for the feed manifest you are about to create must come from somewhere. This is specified in the command help.

`--bzzaccount` is a global option, shared by many commands and could have been set with an environment variable as well. If you provide both, `--user` takes precedence since it is more specific. This behavior is by design. The scope of this PR is to fix something else.

Reply:
In that case it should say specify either `--user` or `--bzzaccount` IMO. If we get confused as swarm devs, no telling what others may feel.

That it's not in the scope of this PR is (still) fine.